apps-contrib/resources/make-it-g8.json

```diff
@@ -0,0 +1,12 @@
+{
+  "mainClass": "com.github.arturopala.makeitg8.MakeItG8",
```

Comment:
Seems the explicit main class is not needed. The right one is found by default:
```text
$ cs launch com.github.arturopala::make-it-g8:latest.stable -- --help

MakeItG8  - convert your project into a giter8 template

Usage: sbt "run [--source {PATH}] [--target {PATH}] [--force] [--name {STRING}] [--package {STRING}] [--description {STRINGURLENCODED}] [--custom-readme-header-path {PATH}] [-K placeholder=textURLEncoded]"

…
```

Reply:
fixed, thanks!